build/cmake/Config.cmake.in

```diff
@@ -0,0 +1,8 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(ZLIB)
```

Comment:
We need `CONFIG` here:
* https://docs.hunter.sh/en/latest/packages/pkg/ZLIB.html

`find_dependency` doesn't support `CONFIG` so just use `find_package` here.